misc/contest/tests/hostname.json

```diff
@@ -0,0 +1,56 @@
+{
+    "JobName": "Hostname validation",
+    "Runs": 1,
+    "RunInterval": "3s",
+    "Tags": ["lscpi"],
+    "TestDescriptors": [
+        {
+            "TargetManagerName": "TargetList",
+            "TargetManagerAcquireParameters": {
+                "Targets": [
+                    {
+                        "FQDN": "10.3.0.88",
```

Comment:
Better perhaps to integrate the FQDN as a parameter which is generated at job creation ?